src/types.ts

```diff
@@ -361,14 +361,42 @@ export interface OperationSuccessResponse {
 
 export interface NetworkConnectionResponse {
     id: string;
-    localChannel: {
-        networkId: string;
-        name: string
-    };
-    remoteChannel: {
-        networkId: string;
-        name: string
-    };
+    status: string;
+    remoteNetworkId: NetworkId;
+    localNetworkId: NetworkId;
+    routingPolicy?: RoutingPolicy;
+}
+
+interface NetworkId {
+    id: string;
+    name: string;
+}
+
+export interface RoutingPolicy {
+    crypto?: RoutingDest;
+    sen?: RoutingDest;
+    signet?: RoutingDest;
+    sen_test?: RoutingDest;
+    signet_test?: RoutingDest;
+}
+
+export interface RoutingDest {
+    scheme: Scheme;
+    dstType: DestType;
+    dstId: string;
+}
+
+export enum Scheme {
+    AUTO = "AUTO",
+    DEFAULT = "DEFAULT",
+    CUSTOM = "CUSTOM",
+}
+
+export enum DestType {
```

Comment:
please indicate that this is a DestType specific for the network connection endpoint (I guess we have another DestType / PeerType already present here which is not the same)

Reply:
@yarinvak, done, thanks!